docs/validators/migrating.md

```diff
@@ -1,75 +1,91 @@
+1. Set and fully sync up a fullnode
+2. Migrate data from old node to new node
+
+## 1. Setup new fullnode
+To avoid as much missed blocks as possible it is recommended that you setup a new server and fully sync up a full node first. To do this,
+you can read the guide [here](../fullnode/setup.md).
+
+Proceed to step 2 only if your new node is no longer catching up.
+
+```
+desmos status 2>&1 | jq "{catching_up: .SyncInfo.catching_up}"
+```
+
+## 2. Migrate Data
+### Stopping both the validator and the newly synced full node
+In order to avoid as much side affects as possible we will be stopping both nodes.
 
 ```
 systemctl stop desmos
 ```
 
-Once you have done so, you need to back up the following data:
+### Backup the following data in the validator node
 
-1. The mnemonic phrase associated with your key.  
-   You should have written it down on a piece of paper when you first created the node.
-   
-2. The validator private key.  
+In order to properly migrate our validator node to another server we will need to backup the following data(s):
+
+1. The validator private key.
    This is located inside the `~/.desmos/config/priv_validator_key.json` file.
-   
-3. The validator consensus state.  
+
+2. The validator consensus state.
    This is located inside the `~/.desmos/data/priv_validator_state.json` file.
+3. If you keep your keys on the node make sure you have the mnemonic phrase associated with your key(s).
 
-:::tip Do not delete your old node yet   
-We highly suggest you to delete your old node instance once that the new node is running properly. This will allow you to recover any information if you forgot to do so.  
-:::
+<!-- :::warning Do not move them in to the new fullnode just yet
+Back them up somewhere save but don't
+::: -->
 
-## 2. Setup your new node
-### Setting up the full node
-Once you are ready to get your new node running, the first thing to do is setup your node as a full node. To do this,
-you can read the guide [here](../fullnode/setup.md).
 
-Once that is done, stop your new node from running by using
+<!-- In order to properly transfer your validator to another server, you first have to stop the running node. To do this, you can execute the following command:
 
 ```
 systemctl stop desmos
-```
+``` -->
+
+<!-- Once you have done so, you need to back up the following data: -->
 
-Now, transfer the following backed up files from the old node into the new one:
+
+:::tip Do not delete your old node yet
+We highly suggest you to delete your old node instance once that the new node is running properly. This will allow you to recover any information if you forgot to do so.
+:::
+
+### Migrating the data in to the fullnode
+With both nodes stopped we will be copying the backedup data in to the newly synced fullnode.
+
+:::warning Warning
+At this point, both nodes should not be running. This is to prevent any possible side effects such as double signing
```

Comment:
Do we have a page explaining `double signing` and the consequence? We can link to it if we do.

Reply:
At the moment, we do not

Reply:
Ok